docs/documentation.rst

```diff
@@ -108,12 +108,12 @@ layout, and use the browser's inspector to look for 404s on static assets.
 
 Publish the new MDN Sphinx Theme
 --------------------------------
-To use the new theme in the public documentation,
+To use the new theme in the public documentation:
 
-1. Commit the new template to ``sphinx-theme``, open a pull request
-2. Remove the clone of the ``sphinx-theme`` from your Kuma repo (Docker method)
-3. After review, merge the PR to the master branch of ``sphinx-theme``
-4. Tag and publish a new version of ``mdn-sphinx-theme`` to PyPI_
-5. Update ``requirements/docs.txt`` in ``kuma``, merge to master
+1. Commit the new template to ``sphinx-theme``, open a pull request.
+2. Remove the clone of the ``sphinx-theme`` from your Kuma repo (Docker method.)
```

Comment:
Hello,

Sorry for my rash requests. I wasn't aware of a writing style guide - my new request is also probably wrong, I am a little lost in the world of contribution. Apologies for the hand holding, but I appreciate your patience.

If you may cancel that request so I can move the period that would be great.

Sorry once again!